src/pages/settings/Subscription/PaymentCard/index.tsx

```diff
@@ -43,18 +44,21 @@ function AddPaymentCard() {
         };
     }, []);
 
-    const addPaymentCard = useCallback((values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_PAYMENT_CARD_FORM>) => {
-        const cardData = {
-            cardNumber: CardUtils.getMCardNumberString(values.cardNumber),
-            cardMonth: CardUtils.getMonthFromExpirationDateString(values.expirationDate),
-            cardYear: CardUtils.getYearFromExpirationDateString(values.expirationDate),
-            cardCVV: values.securityCode,
-            addressName: values.nameOnCard,
-            addressZip: values.addressZipCode,
-            currency: values.currency ?? CONST.PAYMENT_CARD_CURRENCY.USD,
-        };
-        PaymentMethods.addSubscriptionPaymentCard(cardData);
-    }, []);
+    const addPaymentCard = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_PAYMENT_CARD_FORM>) => {
+            const cardData = {
+                cardNumber: CardUtils.getMCardNumberString(values.cardNumber),
+                cardMonth: CardUtils.getMonthFromExpirationDateString(values.expirationDate),
+                cardYear: CardUtils.getYearFromExpirationDateString(values.expirationDate),
+                cardCVV: values.securityCode,
+                addressName: values.nameOnCard,
+                addressZip: values.addressZipCode,
+                currency: values.currency ?? CONST.PAYMENT_CARD_CURRENCY.USD,
+            };
+            PaymentMethods.addSubscriptionPaymentCard(accountID ?? 0, cardData);
```

Comment:
We caused a small bug here - https://github.com/Expensify/App/issues/57122
it is now fixed